fengari.io.conf

```diff
@@ -77,11 +46,10 @@ http {
         }
 
         location /static/ {
-            alias static/;
+           root /var/www/fengari.io;
```

Comment:
fix indentation? (infact the whole file is mixed tabs and spaces)

Reply:
done